docs/extensions/community.md

```diff
@@ -1,6 +1,9 @@
 # Known Extensions List
+
 This file should have all the known Plates extensions listed.
 
+Please feel free submit a [Pull Request](https://github.com/thephpleague/plates/compare) to add your extension to this list.
```

Comment:
the link should just be `https://github.com/thephpleague/plates`

Reply:
done.